src/components/report-filter-controls.tsx

```diff
@@ -68,6 +70,7 @@ function ReportFilterControls({ controls }: ReportFilterControlsProps): VNode {
   }
 
   return (
+    <div>
```

Comment:
Do we need the extra `<div />`?

Reply:
Technically no... I noticed that this whole form was indented wrong so when prettier fixed it, it created a ton of diff noise (made it harder to see that I just added a single section in this PR), so adding that tag let me clean up the diff 😬 

In hindsight I could have recommended viewing with ?w=1 to hide that, I'll remove it